js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetEdges.tsx

```diff
@@ -29,12 +37,16 @@ export const AssetEdges: React.FC<{
         viewportRect={viewportRect}
       />
       <AssetEdgeSet
-        viewportRect={viewportRect}
         color={Colors.Blue500}
-        edges={intersectedEdges.filter(
-          ({fromId, toId}) => highlighted === fromId || highlighted === toId,
+        edges={edges.filter(
```

Comment:
should we filter on `intersectedEdges` instead? There are some special but probably rare cases where it would make a difference.

Reply:
Ahh I think using edges and not intersected edges here ensures that the blue ones don't flicker if you are pan-pan-panning to see where one leads.